pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -1,8 +1,9 @@
-{ mkDerivation, lib, fetchFromGitHub, cmake, ninja, flex, bison, proj, geos
-, xlibsWrapper, sqlite, gsl, qwt, fcgi, python3Packages, libspatialindex
-, libspatialite, postgresql, txt2tags, openssl, libzip, hdf5, netcdf, exiv2
-, protobuf, qtbase, qtsensors, qca-qt5, qtkeychain, qscintilla, qtserialport
-, qtxmlpatterns, withGrass ? true, grass, withWebKit ? true, qtwebkit }:
+{ mkDerivation, lib, fetchFromGitHub, fetchpatch, cmake, ninja, flex, bison
+, proj, geos, xlibsWrapper, sqlite, gsl, qwt, fcgi, python3Packages
+, libspatialindex, libspatialite, postgresql, txt2tags, openssl, libzip, hdf5
+, netcdf, exiv2, protobuf, qtbase, qtsensors, qca-qt5, qtkeychain, qscintilla
+, qtserialport, qtxmlpatterns, withGrass ? true, grass, withWebKit ? true
+, qtwebkit }:
```

Comment:
Instead of reflowing this with every change it should be formatted to be one input per line.